### PR TITLE
AN-301 Update certifi

### PR DESCRIPTION
**Pull request by jgainerdewar**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ run_secret_setup: &run_secret_setup
     command: |
       git clone https://github.com/awslabs/git-secrets.git
       pushd git-secrets >/dev/null
-      git checkout ad82d68ee924906a0401dfd48de5057731a9bc84
+      git checkout 5357e18bc27b42a827b6780564ea873a72ca1f01
```
Comment (LizBaldo) [inline .circleci/config.yml:22]: Could you add a comment explaining what pointing to this specific commit does?

**Review by LizBaldo — APPROVED** [review]
Thanks for doing this, looks like some python setup fun. I have one minor request for adding a comment but should not be a blocker for merging. :shipit:

File: .circleci/config.yml [hunk 2]
```diff
@@ -28,7 +28,7 @@ jobs:
     <<: *job_defaults
     machine:
       # latest pinned 20.04 version https://circleci.com/developer/machine/image/ubuntu-2004
-      image: ubuntu-2004:202201-02
+      image: ubuntu-2004:2024.11.1
```
Comment (aednichols) [inline .circleci/config.yml:31]: Could be worth updating to the [latest LTS](https://circleci.com/developer/machine/image/ubuntu-2404).

20.04 maintenance ends in April 2025.

```suggestion
      image: ubuntu-2404:2024.11.1
```

**Review by aednichols — APPROVED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (jgainerdewar) [reply .circleci/config.yml]: Good call!

**Pull request merged**